src/main/java/seedu/duke/command/ListMenuCommand.java

```diff
@@ -8,6 +8,8 @@
  * Lists all dishes in the menu to the user.
  */
 public class ListMenuCommand extends Command {
+    public static final String COMMAND_WORD = "list menu";
```

Comment:
change the command word to match what we stated in user guide
```suggestion
    public static final String COMMAND_WORD = "list_menu";
```